modules/rte_mis_state/rte_mis_state.module

```diff
@@ -243,3 +272,26 @@ function rte_mis_state_module_implements_alter(&$implementations, $hook) {
     $implementations['rte_mis_state'] = $rte_mis_state_form_alter;
   }
 }
+
+/**
+ * Implements hook_form_ID_alter().
+ */
+function rte_mis_state_form_user_form_alter(&$form, &$form_state, $form_id) {
+
+  $current_user = \Drupal::currentUser();
+  // Get the current user ID.
+  $current_user_id = $current_user->id();
+
+  // Check the current path.
+  $current_path = \Drupal::service('path.current')->getPath();
+
+  // Create a custom link.
+  $user_edit_path = '/user/' . $current_user_id . '/edit';
```

Comment:
I think we should use `user.edit` route to do this.

Reply:
ok